chain/client/Cargo.toml

```diff
@@ -13,7 +13,6 @@ futures = "0.3"
 chrono = { version = "0.4.4", features = ["serde"] }
```

Comment:
strange.. as you're just removing 'serde' from list of deps - maybe this line can be also removed?

Reply:
No, we still use chrono. 

Long explanation:
When we introduced `Clock`, we exported `Instant` from 'std', and `Utc` from `Datetime`, but the migration was only done half way. We never exported `chrono::Datetime` type, nor `chrono::Duration` etc. Right now, half of the import code from our `Clock` crate, half from `chrono`. This needs to be addresses separately.